portfolio/src/main/webapp/graphs.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Graphs</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="https://www.gstatic.com/charts/loader.js"></script>
+    <script src="script.js"></script>
+        <script type="text/javascript">
+
+      // Load the Visualization API and the corechart package.
+      google.charts.load('current', {'packages':['corechart']});
+
+      // Set a callback to run when the Google Visualization API is loaded.
+      google.charts.setOnLoadCallback(drawChart);
+
+      // Callback that creates and populates a data table,
+      // instantiates the pie chart, passes in the data and
+      // draws it.
+      function drawChart() {
+
+        // Create the data table.
+        var data = new google.visualization.DataTable();
+        data.addColumn('string', 'Topping');
+        data.addColumn('number', 'Slices');
+        data.addRows([
+          ['Mushrooms', 3],
+          ['Onions', 1],
+          ['Olives', 1],
+          ['Zucchini', 1],
+          ['Pepperoni', 2]
+        ]);
+
+        // Set chart options
+        var options = {'title':'How Much Pizza I Ate Last Night',
+                       'width':400,
+                       'height':300};
+
+        // Instantiate and draw our chart, passing in some options.
+        var chart = new google.visualization.PieChart(document.getElementById('chart_div'));
+        chart.draw(data, options);
+      }
+    </script>
+  </head>
+  <body>
+      <ul id="navigation-bar">
+          <a class="navigation-bar-items" href="index.html">Home</a>
+          <a class="navigation-bar-items" href="background.html">Background</a>
+          <a class="navigation-bar-items" href="projects.html">Projects</a>
+          <a class="navigation-bar-items" href="image_gallery.html">Image Gallery</a>
+          <a class="navigation-bar-items" href="blog_posts.html">Blog Posts</a>
+          <a class="navigation-bar-items" href="graphs.html">Graphs</a>
+      </ul>

```

Comment:
Nit: Lets indent the ul block to have the same indentation as the div block below to maintain consistency.

Reply:
Fixed indentation.